### ISSUE TITLE
F11 camera is likely malfunctioning (discard batch 6 data)

### ISSUE THREAD

**Issue by kaitlyngaynor** [wildcam-gorongosa/data-cleaning#18]
In the F11 folder from batch 6 (2020 - mid2021), there are a number of photos from May - October 2019 from another camera (likely leftover on SD card). There are only 9 detections from the time period of interest, all from 3 days in February 2021. It seems there was some malfunction in the writing of the SD card.

I'm wondering if it's the camera, because when I downloaded it in October 2021, there were only a handful of detections from 5 days in October. I noted there was likely an issue with the SD card but now I think it's the camera since it had also happened in the previous download.

**We should make sure to swap this camera out ASAP @palme516 .** I have added this to the document Field_notes_KG_Oct2021.docx and will upload version to Drive.

**Comment by raminderbal** [comment]
F11 images from inoperable period in batch 6 not included in MASTER GNP Camera Trap Operation Dates

**Comment by kaitlyngaynor** [comment]
I'm not totally sure what you mean by this but I think we just ignore those 3 days in Feb 2021, the camera clearly wasn't working well

**Comment by raminderbal** [comment]
Yup, Just trying to say that the three days the camera was inoperable were not included in the final operation days
